### DISCUSSION TITLE
Access `pageContext` from within error page

### DISCUSSION THREAD

**Discussion by brillout** [vikejs/vike#849] [Discussion] [answered]
Moved from Discord:

> How do I get the pageContext in the error page (when calling through RenderErrorPage)? 

CC @phiberber

**Comment by brillout** [comment, marked as answer]
Have you tried https://vite-plugin-ssr.com/pageContext-anywhere?

**Reply by phiberber** [reply]
I see. I was trying to access it like one would in a normal page, didn't think It'd need such implementation. 

Thanks!
